automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/TestData.scala

```diff
@@ -1,35 +1,37 @@
 package org.broadinstitute.dsde.firecloud.fixture
 
-object TestData {
+import org.broadinstitute.dsde.workbench.service.Orchestration
 
-  /* PARTICIPANTS AND PARTICIPANT SETS*/
-  object SingleParticipant {
-    val participantEntity = "entity:participant_id\nparticipant1"
-    val entityId = "participant1"
-  }
+case class TestData(prefix: String = "participant") {
```

Comment:
`prefix` is not used so we can revert to `object`

Reply:
`val entityId = Orchestration.randomIdWithPrefix(prefix)`

Reply:
ok that works